### DISCUSSION TITLE
Nous B3Z randomly switching off channel 1 light

### DISCUSSION THREAD

**Discussion by peyn** [Koenkk/zigbee2mqtt#24272] [Devices]
Hi,

I recently bought a Nous B3Z (https://www.zigbee2mqtt.io/devices/B3Z.html#nous-b3z) zigbee switch with 2 channels.
I connected it to the switch in the wall, successfully paired with my zigbee network and it was discovered by the Zigbee2MQTT addon in my Home Assistant installation.

The switch is set to power off the light when it loses and regains the power.
![image](https://github.com/user-attachments/assets/4cf76f8d-f062-470c-942d-91c18dfc5547)

The issue I am facing is that at random intervals the switch switches off the light connected to switch number 1.
Sometimes it's after an hour, sometimes it's after a few minutes (1st & 3rd row is switching off, 2nd & 4th switching on).
![image](https://github.com/user-attachments/assets/725fcc07-6436-48fe-af62-feca81b71f8d)

I checked the wiring at least 3 times already. Tighten every screw. There is no way that something is not connected right.
Could it be a Zigbee2MQTT issue? How can I check? What should I be looking for?

Or is it an issue with the switch? Maybe a known one?

Kind regards,
Mateusz

Edit: I just changed the "Power-on" value to "previous", but the light on switch number 1 was still switched off. It would seem this is not related to loss of power.

**Comment by peyn** [comment]
I removed the device from the Zigbee2MQTT configuration and switched on both lights. They were still on after 3 hours.
The conclusion is that it is related to Zigbee itself, either on the device or Zigbee2MQTT side.

**Comment by peyn** [comment]
I changed the device to another one of the same model - same issue with the switch number 1.

**Comment by ahilwers** [comment]
I have exactly the same problem. It randomly switches off the switch 1 after some time and I have no idea why.

In the Home Assistant log I only can see that it turned off without any trigger but I can't see anything about it in the Log of Zigbee2Mqtt.

<img width="318" alt="Screenshot 2025-01-28 at 19 13 45" src="https://github.com/user-attachments/assets/ce7824b4-b9f0-4aa7-9bf9-0d9febf90d1d" />
